src/core/core.scale.js

```diff
@@ -392,7 +392,7 @@ module.exports = function(Chart) {
 		// Get the correct value. NaN bad inputs, If the value type is object get the x or y based on whether we are horizontal or not
 		getRightValue: function(rawValue) {
 			// Null and undefined values first
-			if (rawValue === null || typeof(rawValue) === 'undefined') {
+			if (rawValue === null || typeof(rawValue) === 'undefined' || rawValue === Infinity) {
```

Comment:
FWIW, we would have to test `-Infinity` as well.

I would use the `isFinite` global function (the check would need to be after the "object" checks, though).